en/guide/index.md

```diff
@@ -22,57 +22,58 @@ As a framework, Nuxt.js comes with a lot of features to help you in your develop
 
 ## How it Works
 
-![Vue with Webpack and Babel](https://i.imgur.com/avEUftE.png)
+![Vue with webpack and Babel](https://i.imgur.com/avEUftE.png)
 
 Nuxt.js includes the following to create a rich web application development:
 
-- [Vue 2](https://github.com/vuejs/vue)
-- [Vue-Router](https://github.com/vuejs/vue-router)
-- [Vuex](https://github.com/vuejs/vuex) (included only when using the [store option](/guide/vuex-store))
-- [Vue-Meta](https://github.com/declandewet/vue-meta)
+- [Vue 2](https://vuejs.org/)
+- [Vue Router](https://router.vuejs.org/en/)
+- [Vuex](https://vuex.vuejs.org/en/) (included only when using the [store option](/guide/vuex-store))
+- [Vue Server Renderer](https://ssr.vuejs.org/en/)
```

Comment:
A suggest to add this one because without SSR, no Nuxt.js ;)

Reply:
I suggest to add:
```
[Vue Server Renderer](https://ssr.vuejs.org/en/)(included only when using `mode`: 'universal')
```
Because Nuxt.js can create web application without server-side rendering :)

@Haeresis

Reply:
Oh it's true!

What do you think about

```
[Vue Server Renderer](https://ssr.vuejs.org/en/) (excluded when using [`mode`: 'spa'](/api/configuration-mode))
```

Because by default store is exclude and Vue SSR is included right?

Reply:
Exactly, it's better like this indeed!